mkat_fpga_tests/test_cbf.py

```diff
@@ -451,19 +450,48 @@ def test_delay_tracking(self):
         # Choose baseline for phase comparison
         baseline_index = baseline_lookup[('m000_x', 'm000_y')]
 
-        correlation_bw = self.corr_freqs.bandwidth
         sampling_period = self.corr_freqs.sample_period
-        test_delays = [0, sampling_period, 1.5*sampling_period]
-
-        for delay in test_delays[0:1]:
-            # set delay on correlator input m000_y
-            # TODO
-            this_freq_dump = self.receiver.get_clean_dump(DUMP_TIMEOUT)
-            data = complexise(this_freq_dump['xeng_raw'][:, baseline_index, :])
-            phases = np.angle(data)
-            plt.plot(self.corr_freqs.chan_freqs, phases)
-            #import IPython; IPython.embed()
-            #plt.show()
+        test_delays = [0, sampling_period, #1.5*sampling_period,
+            3*sampling_period]
+
+        def expected_phases(plot=False):
+            expected_chan_phase = []
+            for channel in self.corr_freqs.chan_freqs:
+                phases = channel * 2 * np.pi * sampling_period
```

Comment:
Where is sampling_period coming from? Perhaps better to pass it into the function?